app/views/index.scala.html

```diff
@@ -183,34 +183,6 @@
             </div>
     </div>
 
-
-
-    <div class="container" id="quotebox-container" style="min-height: 500px;">
```

Comment:
Referring back to [this comment](https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2373#issuecomment-1159321925) on the Github issue: can you go through and make sure to delete any code throughout the codebase that references the things being deleted?

For example, this first HTML element has the id "quotebox-container". If I search the code base for that string, I find 10 lines of CSS in `public/stylesheets/main.css` that are for that ID. Along with 7 lines of code in `public/javascripts/homepage.js` that are for this HTML element.

You'll want to find the "search through all files" type feature in your IDE and use that to search for the various important keywords throughout this HTML, and delete the extra code that you find throughout the rest of the code to keep everything clean!

Some examples of things that you should look into:
* HTML IDs, e.g., `id="quotebox-container"`
* CSS classes, e.g., `class="quotebox"`
* References to the translations of texts on the page, e.g., `Messages("landing.what.people.are.saying")`
* References to images, e.g., `src='@routes.Assets.at("assets/curbed.jpg")`

But if code is being used in another place outside that you aren't deleting here, then it can stay. For example, if you look into the CSS class `im-centered3` below, you'll see that there are other HTML elements on the landing page that use that same CSS class, so you don't want to delete the CSS we have created for it.

The last thing that might be helpful is to let you know about some files that you can completely ignore. These are typically "build" files, so they are just created from our source files. You don't need to take into account code in them, and you shouldn't bother editing them because your changes will just get erased anyway :) Some files to ignore:
* files that end in `.template.scala` (the word template is the one that indicates that we didn't make it)
* Anything with the word "build" in the file path. I don't think you'll run into that here, but there are only a few files like that. Notably `SVLabel.js`, `SVLabel.css`, `Gallery.js`, `Gallery.css`, `SVValidate.js`, `SVValidate.css`, `help.js`, and `Admin.js`

Reply:
Much better! There are a few more things that I found.
* The translations of the text in those sections still exist ("landing.what.people.are.saying", "landing.press.title")
* The CSS class "newslink"
* The images are not being referenced anywhere else but haven't been deleted (technically.png, diamondback.png, bw-background.png, quote.png)